<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1505</width>
    <height>994</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QGridLayout" name="gridLayout">
    <item row="1" column="0">
     <widget class="QVTKOpenGLNativeWidget" name="win_Axial"/>
    </item>
    <item row="1" column="1">
     <widget class="QVTKOpenGLNativeWidget" name="win_Sagittal"/>
    </item>
    <item row="0" column="0">
     <widget class="QPushButton" name="btn_point">
      <property name="text">
       <string>画点</string>
      </property>
     </widget>
    </item>
    <item row="3" column="1">
     <widget class="QVTKOpenGLNativeWidget" name="win_3D"/>
    </item>
    <item row="2" column="1">
     <widget class="QSlider" name="slider_Sagittal">
      <property name="orientation">
       <enum>Qt::Horizontal</enum>
      </property>
     </widget>
    </item>
    <item row="3" column="0">
     <widget class="QVTKOpenGLNativeWidget" name="win_Coronal"/>
    </item>
    <item row="2" column="0">
     <widget class="QSlider" name="slider_Axial">
      <property name="orientation">
       <enum>Qt::Horizontal</enum>
      </property>
     </widget>
    </item>
    <item row="4" column="0">
     <widget class="QSlider" name="slider_Coronal">
      <property name="orientation">
       <enum>Qt::Horizontal</enum>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1505</width>
     <height>23</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <customwidgets>
  <customwidget>
   <class>QVTKOpenGLNativeWidget</class>
   <extends>QOpenGLWidget</extends>
   <header location="global">qvtkopenglnativewidget.h</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
